### PR TITLE
first updated version of the installation/migration documentation

### PR DESCRIPTION
**Pull request by yveskerwyn**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -1,114 +1,221 @@
...
+- [Install JumpScale services on nodes](#jumpscale-nodes)
+- [Deploy virtual machine images](#deploy-images)
+
+
+<a id="prerequisites"></a>
+## Meet the prerequisites
+
+- 3 dedicated controller nodes
+- 10 dedicated CPU nodes
+- 4 dedicated storage nodes
```
Comment (grimpy) [inline docs/Installation.md]: this of course is only 1 of the suipported sizes

File: docs/Installation.md [hunk 2]
```diff
@@ -1,114 +1,221 @@
...
+
+An example of a `system-config.yaml` can be found here: https://github.com/0-complexity/openvcloud_installer/blob/master/scripts/kubernetes/config/system-config.yaml
+
+> **Important** A common technique to create a `system-config.yaml` is to make a copy from another environment and start editing. Please make sure to alter the `ssh.private-key` setting, and not just leave the copy from the other environment.
+
+
+<a id="validate-config"></a>
+## Validate the configuration file
+
+Having valid configuration is off course very important. Validating is done with the OpenvCloud environment manager, a.k.a. **Menaja** (Swahili for 'manager'), available on https://menaja.gig.tech.
```
Comment (grimpy) [inline docs/Installation.md]: with stupid names come stupid typos and ofcourse nobody can remember this anyway its https://meneja.gig.tech/ not menaja

File: docs/Installation.md (hunk 1 above)
Comment (yveskerwyn) [reply docs/Installation.md]: updated worthing

File: docs/Installation.md (hunk 2 above)
Comment (yveskerwyn) [reply docs/Installation.md]: corrected

**Pull request merged**